### ISSUE TITLE
Autolink of "(URL is http://example.com.)" includes the final period

### ISSUE THREAD

**Issue by minux** [vmg/rinku#46]
Autolink of "URL is http://example.com/dir." is:
`URL is <a href="http://example.com/dir">http://example.com/dir</a>.`

whereas autolink of "(URL is http://example.com/dir.)" is
`(URL is <a href="http://example.com/dir.">http://example.com/dir.</a>)`

**Comment by ryanseys** [comment]
Didn't work for your above example either where the last character is a double quote.

`"URL is http://example.com/dir."` --> `"URL is <a href="http://example.com/dir.">http://example.com/dir.</a>"`

**Comment by kivikakk** [comment]
I do see the first example as working, and the second example as failing:

```
>> Rinku.auto_link 'URL is http://example.com/dir.'
=> "URL is <a href=\"http://example.com/dir\">http://example.com/dir</a>."
>> Rinku.auto_link '(URL is http://example.com/dir.)'
=> "(URL is <a href=\"http://example.com/dir.\">http://example.com/dir.</a>)"
```

Will look into this.

**Comment by kivikakk** [comment]
This is fixed as of #61.
